docs/lib/restapi/fragments/flutter/getting-started/11_amplifyInit.md

```diff
@@ -0,0 +1,43 @@
+To start provisioning api resources in the backend, go to your project directory and **execute the command**:
+
+```bash
+amplify add api
+```
+
+Enter the following when prompted:
+```console
+? Please select from one of the below mentioned services: 
+    `REST`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `api`
+? Provide a path (e.g., /book/{isbn}): 
+    `/todo`
+? Choose a Lambda source 
+    `Create a new Lambda function`
+? Provide a friendly name for your resource to be used as a label for this category in the project: 
+    `todo`
+? Provide the AWS Lambda function name: 
+    `todo`
+? Choose the function runtime that you want to use: 
+    `NodeJS`
+? Choose the function template that you want to use: 
+    `Serverless ExpressJS function (Integration with API Gateway)`
```

Comment:
What workflow would you suggest then?  I could talk to the CLI team and ask if they have other suggestions. 

However, I am a bit hesitant because Android and iOS both use this suggestion of NodeJS / Serverless ExpressJS.  Unless we decide to now change those platforms as well, I'm not too sure the point of making our Flutter Rest API different.